src/main/java/entity/Review.java

```diff
@@ -12,11 +12,12 @@ public class Review {
     private int numDislikes;
     private String content;
     private ArrayList<Comment> comments;
+    private int rating;
 
     public Review() {
     }
 
-    public Review(String reviewID, String userID, String company, String content) {
+    public Review(String reviewID, String userID, String company, String content, int rating) {
```

Comment:
why is this constructor necessary? Will we have a review without likes?

Reply:
I don't know, this is how Justin set it up so I just put the rating on with it. I assume he wants a version of the cconstructor without likes for a reason.